src/System.Net.WebHeaderCollection/src/System/Net/HeaderInfo.cs

```diff
@@ -1,29 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-namespace System.Net
```

Comment:
Nice to see the dead code cleaned up. I have to wonder if there were any cases where this isn't dead code however but I don't find any code paths that lead to it from the contract.